test/evaluation.spec.js

```diff
@@ -255,6 +255,13 @@ module.exports.addTests = function({testRunner, expect}) {
       const a = await page.evaluate(() => Array(100 * 1024 * 1024 + 1).join('a'));
       expect(a.length).toBe(100 * 1024 * 1024);
     });
+    it('should throw error with detailed information on exception inside promise ', async({page, server}) => {
+      let error = 'did not catch error';
```

Comment:
nit: `let error = null` to be consistent with our other tests that look at errors

Reply:
I find error messages like 
'Cannot read property 'message' of null'
less informative than
'expect.toContain failed: did not catch error ⊇ Error in promise'

but I agree that the approach should be consistent across the codebase, so changing it to null, we can update the tests later.